addons/mail/static/src/discuss/core/public_web/poll.xml

```diff
@@ -0,0 +1,61 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<templates xml:space="preserve">
+    <t t-name="mail.Poll">
+        <div class="o-discuss-Poll card border-secondary rounded shadow-sm my-2" style="max-width: 500px;">
+            <div class="card-body d-flex flex-column gap-1 p-3">
+                <p class="card-title text-start fw-bold fs-5 mb-0" t-esc="props.poll.question"/>
+                <p class="card-subtitle mb-2 text-muted">
+                    <t t-if="props.allow_multiple_answers">Select one or more answers</t>
+                    <t t-else="">Select one answer</t>
+                </p>
+                <div class="o-discuss-PollAnswer rounded" t-foreach="props.poll.answer_ids" t-as="answer" t-key="answer.id" t-att-style="percentageAttStyle(answer)" t-att-class="{'o-discuss-Poll-showResults': state.showResults, 'o-discuss-Poll-winningAnswer': answer.eq(props.poll.winning_answer_ids[0]) }">
+                    <t t-if="state.showResults" t-call="mail.PollAnswer-showingResults"/>
+                    <t t-else="" t-call="mail.PollAnswer-voting"/>
+                </div>
+                <div class="d-flex align-items-center mt-2">
+                    <p class="small text-muted m-0">
+                        <t t-esc="pollVoteCountText"/>
+                        &#8729;
+                        <t t-if="props.poll.end_message_id">Poll closed</t>
+                        <t t-else="" t-esc="state.remainingTimeText"/>
+                    </p>
+                    <span class="flex-grow-1"/>
+                    <t t-if="!props.poll.end_message_id">
+                        <t t-if="!props.poll.selfAlreadyVoted">
+                            <t t-if="state.showResults">
+                                <button class="btn btn-secondary" t-on-click="toggleShowResults">Go back to vote</button>
+                            </t>
+                            <t t-else="">
+                                <a href="#" class="me-2" t-on-click="toggleShowResults">View results</a>
+                                <button class="btn btn-primary" t-att-disabled="voteButtonDisabled" t-on-click="onClickVote">Vote</button>
+                            </t>
+                        </t>
+                        <button t-else="" class="btn btn-secondary" t-on-click="props.poll.removeVote">Remove Vote</button>
+                    </t>
+                </div>
+            </div>
+        </div>
+    </t>
+    <t t-name="mail.PollAnswer-voting">
+        <label class="o-discuss-PollAnswer d-flex justify-content-between p-2 cursor-pointer rounded border" t-att-class="{
+            'border-primary': state.selectedAnswerIds.has(answer.id),
```

Comment:
`border-primary` with the green checkbox doesn't work visually:
<img width="507" height="223" alt="Image" src="https://github.com/user-attachments/assets/52e74d0a-2399-48af-9a4f-6cd55d097105" />

Green item with purple wrapper (or vice-versa in composer focused + send button we have) looks off.

I think it's good to use `.primary` to highlight action, but we should not overuse it. In this case, I would keep green color as a soft version of primary, with less transparency for making it more visible:
<img width="508" height="222" alt="Image" src="https://github.com/user-attachments/assets/30358170-f818-4179-b067-697241687a86" />